paywall/src/unlock.js/module.ts

```diff
@@ -0,0 +1,68 @@
+import startup from './startup'
+import '../static/iframe.css'
+import { UnlockWindowNoProtocolYet } from '../windowTypes'
+import StartupConstants from './startupTypes'
+
+declare const process: {
+  env: {
+    UNLOCK_ENV: 'dev' | 'test' | 'staging' | 'prod'
+    DEBUG: any
+    PAYWALL_URL: string
+    USER_IFRAME_URL: string
+  }
+}
+
+const constants: { [key: string]: StartupConstants } = {
```

Comment:
Is there any way we can re-use what we already have ... somewhere?

Reply:
We could hoist the common pieces of `index.ts` and `module.ts` to another file and then import in both locations